scanpipe/pipes/d2d.py

```diff
@@ -1694,6 +1694,7 @@ def map_paths_resource(
                 if rel_key not in relations_to_create:
                     relations_to_create[rel_key] = relation
         if paths_not_mapped:
+            to_resource.update(status=flag.REQUIRES_REVIEW)
             to_resource.save()
```

Comment:
Since a save() is triggered right after the update, wouldn't it be better to simply set the value?
`to_resource.status = flag.REQUIRES_REVIEW`